### DISCUSSION TITLE
Can the parent nodes be highlighted and expanded when I click on a node?

### DISCUSSION THREAD

**Discussion by lyleaigbedion** [dowjones/react-dropdown-tree-select#461] [Q&A]
I want users to be able to see the full path after clicking on a node they found in the search or when they click on a node in the tree by highlighting the all the node's parents up to the root parent node within the tree and collapsing all other nodes and paths. Is this possible?

**Comment by mrchief** [comment]
[keepTreeOnSearch](https://github.com/dowjones/react-dropdown-tree-select#keeptreeonsearch) is your friend!
